client/coral-embed-stream/src/components/Comment.js

```diff
@@ -514,19 +462,11 @@ export default class Comment extends React.Component {
                 fill="commentReactions"
                 data={this.props.data}
                 root={this.props.root}
+                asset={asset}
```

Comment:
Passing the `asset` here, we should also let the plugins customize this prop by allowing fragments targeting the `asset`.